src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -1795,6 +1836,9 @@ def dh_parameters_supported(self, p, g):
 
         return codes[0] == 0
 
+    def dhx_serialization_supported(self):
```

Comment:
How about one of the following:
1. dh_seriazliation_with_subgroup_order_supported.
2. dh_RFC5114_serialization_supported.
3. dh_X9.42_serialization_supported.